### DISCUSSION TITLE
I think I might have found a huge exploit

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1328] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
This needs building on, but there is a way to use point blank to redirect pages on chrome://autofill-internals. According to ChatGPT, the page has access to many apis such as chrome.webrequest, used to block sites. This can toggle DNS emulator on a specially crafted page to use opener.chrome to set this up.

**Comment by schoolexploitkid** [comment]
also kinda credit to bypassi
*i cant spell

**Comment by akabutnicer** [comment]
first of all why are we using chat gpt 💀? second of all, does this not require bookmarklets?

**Reply by SlitheringLizard1** [reply]
yeah why chatgpt use copilot

**Reply by akabutnicer** [reply]
imma touch you vro

**Reply by schoolexploitkid** [reply]
im using chatgpt bc i dont have inspect element

**Comment by akabutnicer** [comment]
also i have to add, if the extension in question has a web request listener set up, you can't cancel that via autofill-internals.

**Comment by Dario-git-hub** [comment]
chrome://autofill-internals/ don't have access to any chrome.api (i checked myself)

**Comment by schoolexploitkid** [comment]
it has chrome.send("TurnOffSync"), the goofy one

**Reply by SlitheringLizard1** [reply]
wait can you turn on sync from there

**Comment by jazzymakesstuff** [comment]
would you say this exploit is... massive?

**Reply by SlitheringLizard1** [reply]
ayo you changed your pfp
